recipe/migrations/pypy38.yaml

```diff
@@ -0,0 +1,53 @@
+migrator_ts: 1647123563
+__migrator:
+    migration_number: 1
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.6.* *_73_pypy
+            - 3.7.* *_73_pypy
+            - 3.8.* *_73_pypy
+            - 3.9.* *_73_pypy
+    paused: False
+    longterm: True
+    use_local: False
+    check_solvable: True
+    exclude_pinned_pkgs: False
+    pr_limit: 1
+    bump_number: 1
```

Comment:
Since the builds are purely additive (even if pypy37 still exists on a feedstock, it'll be replaced), do we really need to bump the build_number?

Reply:
Yes, there were other changes like compiler version updates, numpy version bump etc that changes the hash and we don't want to upload a new package with same build number and different hash for cpython.